sdk/ml/azure-ai-ml/azure/ai/ml/entities/_endpoint/serverless_endpoint.py

```diff
@@ -0,0 +1,80 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+# pylint: disable=unused-argument
+
+from os import PathLike
+from typing import Any, AnyStr, Dict, IO, Optional
+
+from azure.ai.ml._utils._experimental import experimental
+from azure.ai.ml.constants._common import KEY
+from azure.ai.ml._restclient.v2024_01_01_preview.models import (
+    ServerlessEndpoint as RestServerlessEndpoint,
+    ServerlessEndpointProperties as RestServerlessEndpointProperties,
+    ModelSettings as RestModelSettings,
+    Sku as RestSku,
+)
+
+from .endpoint import Endpoint
+
+
+@experimental
+class ServerlessEndpoint(Endpoint):
```

Comment:
This model seems to be a duplicate of the generated-and-patched one in `_autogen_entities`?

Reply:
yes, thanks for pointing that out. I'll remove it

Reply:
removed